Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1480,7 +1487,7 @@ extension BrowserViewController: URLBarDelegate {
     func urlBarDisplayTextForURL(url: NSURL?) -> String? {
         // use the initial value for the URL so we can do proper pattern matching with search URLs
         var searchURL = self.tabManager.selectedTab?.currentInitialURL
-        if searchURL == nil || ErrorPageHelper.isErrorPageURL(searchURL!) {
+        if searchURL == nil || searchURL!.isErrorPageURL() {
```

Comment:
While here, might as well do this properly: `if searchURL?.isErrorPageURL() ?? true`